tests/phpunit/tests/test-multisite.php

```diff
@@ -1,13 +1,20 @@
 <?php
 
-if ( is_multisite() ) :
+class Multisite_Test extends PLL_UnitTestCase {
 
-	class Multisite_Test extends PLL_UnitTestCase {
-		public function test_new_site() {
-			$site_id = self::factory()->blog->create();
-			$options = get_option( 'polylang' );
-			$this->assertNotFalse( $options );
+	public function set_up() {
+		if ( ! is_multisite() ) {
+			$this->markTestSkipped( 'This test requires WordPress multisite.' );
```

Comment:
I would prefer not to change this. At least not as proposed and not in this PR.

The reason it is written like this is to avoid skipped tests for a normal install (to avoid the message "OK, but incomplete, skipped, or risky tests!" when testing the stable version with the the stable WordPress. Yes I like green ;)

However, I am ok to improve this but:
1. We must do this in a separate PR
2. All mulitisite test classes must be modified for consistency
3. I prpose to create a multisite group which we could exclude by default and include when testing a multisite.

Reply:
Done in [131c8f0](https://github.com/polylang/polylang/pull/1319/commits/131c8f0a584c7a5dc3ee54cb6c0eb68ba7f8ba9c).